graylog2-server/src/main/java/org/graylog2/indexer/counts/Counts.java

```diff
@@ -50,14 +62,25 @@ private long totalCount(final String[] indexNames) {
             return 0L;
         }
 
+        final List<String> indices = Arrays.asList(indexNames);
+        final String query = new SearchSourceBuilder()
+                .query(QueryBuilders.matchAllQuery().buildAsBytes(XContentType.JSON))
```

Comment:
Isn't it enough to just call `.query(QueryBuilders.matchAllQuery())` or is it important to pass a `BytesReference`?